src/store/workflows.module.js

```diff
@@ -195,9 +188,8 @@ function addChild (parentNode, childNode) {
 
 /* Remove a child node from a parent node. */
 function removeChild (state, node, parentNode = null) {
-  // console.log(`$t -- ${node.id}`)
   let key = 'children'
-  if (node.type === '$namesapce') {
+  if (node.type === '$namespace') {
```

Comment:
⚠️  Not a console log change, but think this needs correcting.

Reply:
The tests weren't failing for this which is concerning, it would be worth adding a test which attempts to remove a namespace child.

It might be as simple as:

```js
store.commit('workflows/UPDATE', { id: '~u/w//$namespace|foo' })
store.commit('workflows/REMOVE', '~u/w//$namespace|foo' )
```